app/views/course/leaderboards/show.html.slim

```diff
@@ -18,7 +18,7 @@ div.col-sm-12.col-md-6
               = link_to_course_user(course_user) do
                 = format_inline_text(course_user.name)
             div
-              = t('.level', level_number: course_user.level_number)
+              = t('.level', level_number: @course.level_number_for(course_user.experience_points))
```

Comment:
Currently it isn't, but I can explore to see whether I can make it a calculated attribute. 

Reason I did it in this manner instead was because: 
i) there is some logic to derive the number, ie. if experience_points is below zero, level is 0, and
ii) I'm not sure if I'm able to precompute as the `level_number` itself is a calculated attribute on each level.